Cargo.lock

```diff
@@ -704,9 +705,9 @@ checksum = "61807f77802ff30975e01f4f071c8ba10c022052f98b3294119f3e615d13e5be"
 
 [[package]]
 name = "num-complex"
-version = "0.4.2"
+version = "0.4.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "7ae39348c8bc5fbd7f40c727a9925f03517afd2ab27d46702108b6a7e5414c19"
+checksum = "1ba157ca0885411de85d6ca030ba7e2a83a28636056c7c699b07c8b6f7383214"
 dependencies = [
  "num-traits",
```

Comment:
@brycemines , you might need to include `num-traits`, since it's a dependency.

Reply:
What would I need to do to include it? I see that `num-traits` is listed on line 726, but is that something different?

Reply:
Oh, I missed that. So it's all right. Thanks!